visualization/app/codeCharta/ui/dialog/dialog.changelog.component.ts

```diff
@@ -35,7 +36,12 @@ export class DialogChangelogController {
 				const start = lineIndex + 2
 				const end = this.findEndChangesLine(changelogLines, lineIndex)
 				for (const changeLine of changelogLines.slice(start, end)) {
-					changelogTypes.push(changeLine)
+					if (!changeLine.includes("<img")) {
+						const newChangeLine = `${changeLine}<br>`
+						changelogTypes.push(newChangeLine)
+					} else {
+						changelogTypes.push(`${changeLine}<br>`)
+					}
```

Comment:
Hm, does this have any effect? :-)

Reply:
yes otherwise the images would look bit ugly
![Screenshot 2022-01-17 at 16 06 12](https://user-images.githubusercontent.com/57844849/149817150-ac761b90-87a0-4eac-9263-741ad94b8eca.png)
.

Reply:
But dont‘t you do the same Thing (adding  a Break tag) in Borg cases? Then, no of Would be neccessary.

Reply:
you are right. :)